src/Commands/Server/ToolOperations.cs

```diff
@@ -95,7 +95,7 @@ private async ValueTask<CallToolResponse> OnCallTools(RequestContext<CallToolReq
         var commandContext = new CommandContext(_serviceProvider);
 
         var args = parameters.Params.Arguments != null
-            ? string.Join(" ", parameters.Params.Arguments.Select(kvp => $"--{kvp.Key} \"{kvp.Value}\""))
+            ? string.Join(" ", parameters.Params.Arguments.Select(kvp => $"--{kvp.Key} \"{(kvp.Value).ToString().Replace("\"", "'")}\""))
```

Comment:
from the teams chat, this breaks

```kusto
| where something has "'"
```

please add this as another unit test in the PR